docs/populating-configuration.md

```diff
@@ -0,0 +1,353 @@
+> NOTE some step in this document might refer to Red Hat internal components
+  that you do not have access to
+
+# Populating Configuration
+The document describes how to prepare Fleet Manager to be able to start by
+populating its configurations.
+
+For convenience, the following Makefile target is provided:
+
+```
+make secrets/setup/empty
+```
+
+This target initializes empty files that are necessary for the service to boot.
+It can be used to quickly get the service running locally, although it does not
```

Comment:
This change does not seem correct to me. With the new suggested change would be:
`It can be used to quickly to get ...`.

It should either be `It can be used to quickly get ...` or `It can be used quickly to get the ... `

Reply:
misread this, didn't see the `to` at the start. can ignore this one 👍